model/task/task.go

```diff
@@ -1313,16 +1313,16 @@ func (t *Task) UpdateHeartbeat() error {
 	)
 }
 
-// Blacklist sets the priority of a task so it will never run.
+// Block sets the priority of a task so it will never run.
 // It also deactivates the task and any tasks that depend on it.
 // The build cache is not updated
-func (t *Task) Blacklist(user string) ([]Task, error) {
-	t.Priority = evergreen.BlacklistPriority
+func (t *Task) Block(user string) ([]Task, error) {
```

Comment:
this seems to be overloading the Blocked terminology
(until now it's just been for tasks with unsatisfied dependencies)

Reply:
Agreed, can we call them disabled tasks?

Reply:
fixed